collect_app/src/main/java/org/odk/collect/android/instancemanagement/InstanceSubmitter.kt

```diff
@@ -119,8 +118,9 @@ class InstanceSubmitter(
     }
 
     private fun logUploadedForm(instance: Instance) {
-        val action = if (isGoogleSheetsProtocol()) "HTTP-Sheets auto" else "HTTP auto"
-        val label = Collect.getFormIdentifierHash(instance.formId, instance.formVersion)
-        analytics.logEvent(AnalyticsEvents.SUBMISSION, action, label)
+        val key = if (isGoogleSheetsProtocol()) "HTTP-Sheets auto" else "HTTP auto"
+        val value = Collect.getFormIdentifierHash(instance.formId, instance.formVersion)
+
+        Analytics.log(AnalyticsEvents.SUBMISSION, key, value)
```

Comment:
Maybe we can just make this a "form" event now (again because we're not going to continue Google Drive support)?